### ISSUE TITLE
Cannot exclude models from generation

### ISSUE THREAD

**Issue by raymclee** [paljs/prisma-tools#221]
I have below pal.js, when I use pal -g. it still generate all model

```
module.exports = {
  backend: {
    generator: "nexus",
    onDelete: true,
    output: "lib/api/graphql",
    models: ["User", "Shop", "Order", "OrderItem", "Table", "Product"],
    excludeFieldsByModel: { User: ["accounts", "sessions"] },
  },
};
```

**Comment by AhmedElywa** [comment]
@raymclee our cli not delete your old files if you delete all `output: "lib/api/graphql",` content and run again will generate the models you put in the pal.js file

**Comment by raymclee** [comment]
I have tried to change the output to new folder and it still generate all model. 

excludeModels also not working but "excludeFieldsByModel" works

**Comment by AhmedElywa** [comment]
Yes, you are right. working on a fix now

**Comment by raymclee** [comment]
> Yes, you are right. working on a fix now

many thanks!

**Comment by AhmedElywa** [comment]
Also, you can use  `pal generate User,Shop,Order,OrderItem,Table,Product`

**Comment by raymclee** [comment]
> Also, you can use `pal generate User,Shop,Order,OrderItem,Table,Product`

yeah, that works thanks.

is it possible to get the generated file in lowercase?

**Comment by AhmedElywa** [comment]
> is it possible to get the generated file in lowercase?

It's hard and will make a breaking change.

you can build a small script to run after our generator to select all folder and change their names 
Don't forget to change imports in the index file

**Comment by raymclee** [comment]
> > is it possible to get the generated file in lowercase?
> 
> It's hard and will make a breaking change.
> 
> you can build a small script to run after our generator to select all folder and change their names
> Don't forget to change imports in the index file

got it. thanks!

**Comment by AhmedElywa** [comment]
this cli issue now fixed in the last version `3.4.0`

**Comment by Frosty21** [comment]
is `excludeQueriesAndMutationsByModel` working? could you show and example. im trying but it seems like its not working
within my pal.js file
```
module.exports = {
  backend: {
    generator: 'nexus',
    output: 'src/server/graphql',
    models: ["User"],
    excludeQueriesAndMutationsByModel: [{ modelName: "User", QueriesAndMutations: ['updateMany', 'deleteMany', 'upsertOne'] }]
  },
  frontend: {
    admin: true,
  },
};
